app/mailers/claim_mailer.rb

```diff
@@ -13,7 +13,7 @@ def approved(claim)
 
   def rejected(claim)
     @claim_description = claim_description(claim)
-    @possible_rejection_reasons = I18n.t("#{claim.policy.routing_name.underscore}.possible_rejection_reasons", qts_year: ineligible_qts_year(claim))
+    @possible_rejection_reasons = I18n.t("#{claim.policy.locale_name}.possible_rejection_reasons", qts_year: ineligible_qts_year(claim))
```

Comment:
"locale name" makes me think that it will be the name of a locale e.g. `en`, `es`, `pt`. Something that makes it clear it's a key in the locales file might be better, e.g. `i18n_key`.

Reply:
How about `locale_key`?

Reply:
Works for me 👍